eng/scripts/Generate-APIView-CodeFile.ps1

```diff
@@ -0,0 +1,49 @@
+param (
+  [Parameter(mandatory = $true)]
+  $ArtifactPath,
+  $NpmDevopsFeedRegistry = "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js/npm/registry/"
+)
+
+Set-StrictMode -Version 3
+if (!(Test-Path -Path $ArtifactPath))
+{
+  Write-Error "Incorrect path to api-extractor artifacts. Path: $($ArtifactPath)"
+  exit 1
+}
+
+$apiviewParser = "@azure-tools/ts-genapi@1.0.3"
```

Comment:
Do we want to hardcode this version in this script? Do we have this anywhere else that we should try to share?

Reply:
I can remove the version from here. It's not really required. We will release a new version to devops only when we want to get API reviews updated using latest version.

Reply:
If removing the version implies that we always take the latest then I also don't think that is a good idea because we need to try and remain consistent and repeatable. My main concern is we need to have a plan in place to update this version and if we have it in multiple places we should try to share it.